src/pages/ThorChainLP/components/AddLiquidity/AddLiquidityInput.tsx

```diff
@@ -549,11 +549,14 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
   }, [poolAsset])
 
   const memo = useMemo(() => {
-    if (!runeAccountId) return undefined
+    const accountId = runeAccountId || poolAssetAccountId
+    if (!accountId) return ''
```

Comment:
Failsafe just in case - this will wrongly estimate with empty `address` part in the memo, but we should always have in the other

Reply:
Any reason we don't return `undefined` here?

Reply:
As mentioned above, just a failsafe in case things are loading - we *should* have both but IMO safer that way - happy to still return `undefined` however if that's deemed clearer/cleaner !